spin-up-hub/dist/index.html

```diff
@@ -40,6 +40,8 @@
       rel="stylesheet"
     />
 
+    <title>Spin up Hub</title>
```

Comment:
I _think_ we usually refer to this with title-case

```suggestion
    <title>Spin Up Hub</title>
```